src/components/team-member-entries/TeamMemberEntries.tsx

```diff
@@ -32,9 +53,18 @@ export const TeamMemberEntries = ({ isModalActive, handleModal }: Types.AtBuildT
             <option value={entry} label={entry} key={entry} />
           ))}
         </Styled.Select>
+        <Styled.Label>Sort by name:</Styled.Label>
+        <Styled.Select value={sortOrder} onChange={(e) => setSortOrder(e.target.value)}>
+          <option value="">--Select option--</option>
+          {sortOptions.map((label) => (
+            <option key={Math.random()} label={label} value={label}>
+              {label}
+            </option>
+          ))}
```

Comment:
Replace `Math.random()` with `label + i`: 
```suggestion
          {sortOptions.map((label, i) => (
            <option key={label + i} label={label} value={label}>
              {label}
            </option>
          ))}
```